setup.py

```diff
@@ -38,7 +38,7 @@
     'lxml',
     'scikit-rf',
     'BeautifulSoup4',
-    'justag==0.0.4.2',
+    'justag==0.0.4.4',
```

Comment:
updated the JusTAG version to resolve an issue having to due with a limit on the total number of registers.